client/src/lib/components/DatasetForm/DatasetForm.svelte

```diff
@@ -543,40 +453,22 @@
   <h2 id="ouverture" class="fr-mt-6w fr-mb-5w">Ouverture</h2>
 
   <div class="form--content fr-mb-8w">
-    <div
-      class="fr-input-group fr-my-4w"
-      class:fr-input-group--error={$errors.publishedUrl}
+    <InputField
+      name="publishedUrl"
+      value={$form.publishedUrl}
+      error={$errors.publishedUrl}
+      on:input={handleFieldChange}
+      on:blur={handleFieldChange}
     >
-      <label class="fr-label" for="publishedUrl">
-        Page open data
-        <span class="fr-hint-text" id="publishedUrl-desc-hint">
-          Si le jeu de données est publié en open data, saisissez ici le lien de
-          la page web associée.
-        </span>
-      </label>
-
-      <input
-        class="fr-input"
-        class:fr-input--error={$errors.lastUpdatedAt}
-        aria-describedby={$errors.publishedUrl
-          ? "publishedUrl-desc-error"
-          : null}
-        type="text"
-        id="publishedUrl"
-        name="publishedUrl"
-        on:input={handleFieldChange}
-        bind:value={$form.publishedUrl}
-      />
-
-      {#if $errors.publishedUrl}
-        <p id="publishedUrl-desc-error" class="fr-error-text">
-          {$errors.publishedUrl}
-        </p>
-      {/if}
-    </div>
+      <svelte:fragment slot="label">Page open data</svelte:fragment>
+      <svelte:fragment slot="hintText">
+        Si le jeu de données est publié en open data, saisissez ici le lien de
+        la page web associée.
+      </svelte:fragment>
+    </InputField>
   </div>
 
-  <div class="fr-input-group button--container fr-mb-6w">
+  <div class="fr-grid-row fr-grid-row--right fr-mb-6w">
```

Comment:
Attention je ne pense pas que c'est le bon endroit pour utiliser la classe fr-grid-row.

Pour moi les classe fr-grid-row et fr-grid-column sont à utiliser pour structurer une page et non une partie d'une page et elle doivent être inclues dans un container ayant la class `fr-container` comme le montre cet exemple (issu de la [doc](https://gouvfr.atlassian.net/wiki/spaces/DB/pages/222331372/Grille+et+points+de+rupture+-+Grid+and+breakpoints)) :

```
<div class="fr-container">
  <div class="fr-grid-row">
    <div class="fr-col"></div>
    <div class="fr-col-4"></div>
    <div class="fr-col-12 fr-col-lg-4"></div>
  </div>
</div>
```

Reply:
Hmm effectivement les exemples du DSFR ne montrent un usage de `fr-grid-row` qu'à l'intérieur d'un `fr-container` Je vais retirer ça, d'ailleurs ça n'aurait pas dû être là cette modif.